docs/usage/docker-shim-removal.md

```diff
@@ -0,0 +1,47 @@
+# Kubernetes dockershim removal
+
+## What's happening?
+With Kubernetes v1.20 the built-in dockershim [was deprecated](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#dockershim-deprecation) and is scheduled to be [removed with v1.24](https://github.com/kubernetes/enhancements/issues/2221).
+Don't Panic! The Kubernetes community has [published a blogpost](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) and an [FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/) with more information.
+
+Gardener also needs to switch from using the built-in dockershim to `containerd`.
```

Comment:
```suggestion
Gardener also needs to switch from using the built-in dockershim to alternative container runtime interface and this is `containerd`.
```

Reply:
Can you help me understand what this change is about? Do you want to express that there's no real requirement to switch to `containerd`, but rather a decision we made? For example that we could have switched to `cri-o`?

Reply:
More or less yes. Gardener is extensible and currently we support two different CR(I)s - docker and containerd. We want to get rid of the former in favor of the later. But in the future, we can indeed support other CR(I)s. My proposal is just to change a bit the wording to not make hard statement that only containerd is and will be supported (at least this is my personal impression when I read it).

Reply:
Yeah, I even included this to communicate clearly that we don't have any plans to do anything else at the moment.
> At this point in time, we have no plans to support other container runtimes, such as `cri-o`

I do understand that you want to communicate that technically we could also support other runtimes or could have chosen `cri-o` instead of `containerd`, but for this purpose I considered this discussion out of scope. Here the point was: upstream k8s changes and therefore also gardener needs to change.

Do you want me to change the sentence to a more nuanced version or is it fine with you to keep it like this?

Reply:
Having in mind the additional details later in the text, I am fine also how it is currently.